.github/workflows/release.workflow.yml

```diff
@@ -0,0 +1,17 @@
+name: Publish to npm
+
+on: [release]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12"
+          registry-url: "https://registry.npmjs.org"
+      - run: npm install
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Where are stored the secrets?

Reply:
There are stored per repository in `Settings -> Secrets` but I'll update the Github billing plan. So we could store them directly in the organization secrets and use them on all the repositories